### PR TITLE
Pin rules_closure to commit c56b953 

### PR DESCRIPTION
**Pull request by yatbear**
Pin to https://github.com/bazelbuild/rules_closure/pull/599 in order to unblock https://github.com/tensorflow/tensorboard/pull/6721. Note that this `rule_closure` commit uses a specific [`rules_webtesting` version](https://github.com/bazelbuild/rules_closure/blob/c56b953045c883b393d6f0f5dbf9900df71949c1/closure/repositories.bzl#L1099), which has a `go_register_toolchains` version conflict with `grpc_extra_deps`:
- `rules_webtesting` specifies version [`1.20.5`](https://github.com/bazelbuild/rules_webtesting/blob/9e9361ba887a3b687f537c02409b690b62fecdfe/WORKSPACE#L42)
- `grpc_extra_deps` uses version [`1.18`](https://github.com/grpc/grpc/blob/master/bazel/grpc_extra_deps.bzl#L55)

The only thing we need that is installed by `grpc_extra_deps` is `rules_swift`, added `build_bazel_rules_swift` directly in order to remove `grpc_extra_deps`.

Googlers, see cl/597800285 and b/327605070 for context.

#oncall

### CODE REVIEW

**Comment by yatbear** [comment]
cc @comius @gkdn

**Review by rileyajones — APPROVED** [review]

File: WORKSPACE [hunk 1]
```diff
@@ -55,11 +55,13 @@ py_repositories()
 
 http_archive(
     name = "io_bazel_rules_closure",
-    sha256 = "6a900831c1eb8dbfc9d6879b5820fd614d4ea1db180eb5ff8aedcb75ee747c1f",
```
Comment (arcra) [inline WORKSPACE:58]: Thanks, Yating!

One question... how do we get these values? Should we just comment them out? Or is it easy to return to this if we need to?

**Review by arcra — APPROVED** [review]

File: WORKSPACE [hunk 2]
```diff
@@ -26,10 +26,11 @@ versions.check(
 
 http_archive(
     name = "io_bazel_rules_webtesting",
-    sha256 = "9bb461d5ef08e850025480bab185fd269242d4e533bca75bfb748001ceb343c3",
+    sha256 = "41d500a97ad9621dcf92fcb0cd77916e517388b196e5c3f0e63c7753e983b2bb",
+    strip_prefix = "rules_webtesting-4d7ec75d1cbb289f977b41638fc8b630bdf22bee",
     urls = [
-        "http://mirror.tensorflow.org/github.com/bazelbuild/rules_webtesting/releases/download/0.3.3/rules_webtesting.tar.gz",
-        "https://github.com/bazelbuild/rules_webtesting/releases/download/0.3.3/rules_webtesting.tar.gz",
+        "http://mirror.tensorflow.org/github.com/bazelbuild/rules_webtesting/archive/4d7ec75d1cbb289f977b41638fc8b630bdf22bee.tar.gz",
```
Comment (arcra) [inline WORKSPACE:32]: I think all of the issues we encountered with `platform_http_file` and `web_test_archive` rules being removed are due to this change.

But I don't understand why we're changing this reference... I thought we only had to pin the one below, related to "closure" rules.

And I don't understand what version/snapshot we are referencing here. (Maybe we should add a comment?)

I suppose it's fine to update it, and we'll need to make these changes whenever we do, anyway... But I think we could make these changes independently. Or is there a reason to make them together?

File: WORKSPACE (hunk 1 above)
Comment (arcra) [reply WORKSPACE:58]: Ok, but how about the urls? Where/how did you find this new value to reference? Is there a guide for how to update these references?

Is it just that these particular versions are the latest ones available at http://mirror.tensorflow.org?

**Pull request closed without merging**